package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@hapi/jwt",
     "description": "JWT (JSON Web Token) Authentication",
-    "version": "2.0.0",
```

Comment:
all of the other changes look good, we typically only update the version on a release commit, can you revert this to 2.0.0 for now.

Reply:
This was due to a rebase after I initially forked it.  The master branch is now on 2.0.1.  Perhaps I did not do my rebase in the correct fashion.